examples/low-level-api/create_vc.rs

```diff
@@ -37,17 +42,43 @@ pub async fn create_vc() -> Result<()> {
 
   println!("Credential JSON > {:#}", credential);
 
-  // Validate the verifiable credential
+  // Before passing this credential on the issuer wants to validate that some properties
+  // of the credential satisfy their expectations.
+  // In order to validate a credential the issuer's DID Document needs to be resolved.
+  // Since the issuer wants to issue and verify several credentials without publishing updates to their DID Document
+  // the issuer decides to resolve their DID Document up front now so they can re-use it.
+
   let resolver: Resolver = Resolver::new().await?;
-  let validation: CredentialValidation = common::check_credential(&resolver, &credential).await?;
-  println!("Credential Validation > {:#?}", validation);
-  assert!(validation.verified);
+  let resolved_issuer_doc: ResolvedIotaDocument = resolver.resolve(issuer_doc.id()).await?;
+  // Validate the credential's signature using the issuer's DID Document, the credential's semantic structure,
+  // that the issuance date is not in the future and that the expiration date is not in the past:
+  CredentialValidator::validate(
+    &credential,
+    &CredentialValidationOptions::default(),
+    &resolved_issuer_doc,
+    FailFast::Yes,
+  )?;
+  // The issuer is now sure that the credential they are about to issue satisfies their expectations
+  // hence the credential is now serialized to JSON before passing it to the subject in a secure manner.
+  // This means that the credential is NOT published to the tangle where it can be accessed by anyone.
+  let credential_json: String = credential.to_json()?;
```

Comment:
This should use the `Resolver::verify_credential` function, if at all.

Reply:
`Resolver::verify_credential` is used later in the example. My point here was to illustrate that an issuer might want to check the credentials they issue (as a sanity check), but they would probably not want to resolve their DID Document every time.

Reply:
They would not need to if they `Resolver::verify_credential` allowed passing in their document. This point may be moot if `verify_credential` is removed.